### PR TITLE
[Internal] Fixed test StringFormatNonStringType failed, when current culture is not en-US or tr-TR.

### PR DESCRIPTION
**Pull request by workgroupengineering**
### Description of Change ###

Using InvariantCulture instead of CurrentCulture.

### Bugs Fixed ###

- Provide links to bugs here

### API Changes ###

none

### Behavioral Changes ###
none

### PR Checklist ###

- [x] Has tests (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [x] Consolidate commits as makes sense

### CODE REVIEW

**Comment by rmarinho** [comment]
build

**Comment by rmarinho** [comment]
build

File: Xamarin.Forms.Core.UnitTests/BindingBaseUnitTests.cs [hunk 1]
```diff
@@ -166,17 +166,25 @@ public void ChangeAfterApply()
...
-			bo.SetBinding(property, binding);
+				var property = BindableProperty.Create("Foo", typeof(string), typeof(MockBindable));
+				var binding = new Binding("Value", stringFormat: "{0:P2}");
+
+				var vm = new { Value = 0.95d };
+				var bo = new MockBindable { BindingContext = vm };
+				bo.SetBinding(property, binding);
 
-			if (System.Threading.Thread.CurrentThread.CurrentCulture.Name == "tr-TR")
-				Assert.That(bo.GetValue(property), Is.EqualTo("%95,00"));
```
Comment (StephaneDelcroix) [inline Xamarin.Forms.Core.UnitTests/BindingBaseUnitTests.cs]: that part of the test is now missing

Comment (workgroupengineering) [reply Xamarin.Forms.Core.UnitTests/BindingBaseUnitTests.cs]: Hi , thanks for your review.
I had understood that the test StringFormatNonStringType wold to check if StringFormat behavior is correct with non-string values indifferently of the culture.

What do you think if I use [TestCaseSource](https://github.com/nunit/docs/wiki/TestCaseData) to test the behavior for each culture?

**Comment by rmarinho** [comment]
build

**Pull request closed without merging**